test/e2e/kubectl.go

```diff
@@ -1695,6 +1695,50 @@ metadata:
 			}
 		})
 	})
+
+	framework.KubeDescribe("Kubectl v1.6 deployment reaper", func() {
+		var nsFlag string
+		var name string
+
+		BeforeEach(func() {
+			nsFlag = fmt.Sprintf("--namespace=%v", ns)
+			name = "e2e-test-nginx-deployment"
+		})
+
+		AfterEach(func() {
+			By("deleting the deployment " + name)
+			framework.RunKubectlOrDie("delete", "deployment", name, nsFlag)
+
+			By("verifying replicasets were deleted")
+			rsList, err := c.Extensions().ReplicaSets(ns).List(metav1.ListOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			Expect(rsList.Items).To(HaveLen(0))
+		})
+
+		// Question: should we include "Conformance" label? It is an explicit 1.5-1.6 version skew test.
```

Comment:
I will remove this comment after obtaining reply.
For now, i do not put the `Conformance` label as it is an explicit 1.5-1.6 version skew test.